CHANGELOG.md

```diff
@@ -24,14 +24,15 @@ release.
 <!--lint disable maximum-line-length-->
 <table>
 <tr>
-  <th title="Current"><a href="doc/changelogs/CHANGELOG_V14.md">14</a><sup>Current</sup></th>
+  <th title="LTS Until 2023-04"><a href="doc/changelogs/CHANGELOG_V14.md">14</a><sup>LTS</sup></th>
   <th title="Current"><a href="doc/changelogs/CHANGELOG_V13.md">13</a><sup>Current</sup></th>
```

Comment:
631dd21709 removed the column for 13.x releases but didn't remove this table header. PR to fixup in https://github.com/nodejs/node/pull/35756.